docs/pages/ui-programming/react-native-toast.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How to display a toast in React Native
+sidebar_title: Displaying toast messages
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+## What is a toast?
+
+According to the [Android Developers
```

Comment:
Something about why you might use a toast: Toasts are the standard technique to notify your users about something without interrupting what they are doing.